lips/lip-1.mdx

```diff
@@ -498,19 +452,106 @@ Valid values are:
 * `needs_kyc_data` - KYC data about the subaddresses is required by this actor.
 * `needs_recipient_signature` - Can only be associated with the sender actor.  Means that the sender still requires that the recipient VASP provide the signature so that the transaction can be put on-chain.
```

Comment:
we said remove needs_recipient_signature, right?

Reply:
Its gone :)